### DISCUSSION TITLE
ballooning assembly as I add more coverage

### DISCUSSION THREAD

**Discussion by waalkes** [ablab/spades#746] [Q&A]
I am paired end assembling a known genome - Salmonella enterica strain LT2 (CP014051.2) ~4.9M in size

I am assembling at 20x,50x,100x,300x and 600x coverage.

The results I am getting are strange. I am assembling with default parameters:
./spades.py       --pe1-1  LT2_DS20_S99_L001_R1_001.fastq.gz    --pe1-2 LT2.highdepth/LT2_DS20_S99_L001_R2_001.fastq.gz    -o      /home/local/AMC/waalkes/LT20

As I increase coverage the assembly size increases dramatically.  Here are the numbers.  
20x - 5.3M
50x - 5.9M
100x - 7.0M
300x - 10.9
600x - 14.2M 

Any help?  Thanks

**Comment by asl** [comment]
I would suggest to check for possible contamination.

**Comment by waalkes** [comment]
I don't think that is the problem.  Same libraries work fine with ABYSS and masurca and when I align and variant call with them.

**Comment by asl** [comment]
Well, SPAdes assembles whatever you're having in your input reads. It does not do any coverage filtering like other assemblers do. The typical source of such increase of assembly size is either contamination or some strain variability within input data. In any case, since you're having the output you could easily check what's inside.
